libraries/ESP8266WiFi/src/WiFiClient.cpp

```diff
@@ -42,6 +42,8 @@ extern "C"
 #include "c_types.h"
 
 uint16_t WiFiClient::_localPort = 0;
+bool WiFiClient::_defaultNoDelay = true;
```

Comment:
I suggest a big fat comment here about what the four combinations of nodelay and sync do, because I can't for the life of me keep them straight in my head.